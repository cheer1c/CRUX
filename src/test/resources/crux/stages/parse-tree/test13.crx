// test returning int

func myInt() int {
  return readInt();
}

func main() {
  printInt(myInt());
}
