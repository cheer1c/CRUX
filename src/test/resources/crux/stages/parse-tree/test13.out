program
  declList
    decl
      funcDefn
        paramList
        type
        stmtBlock
          stmtList
            stmt
              returnStmt
                expr0
                  expr1
                    expr2
                      expr3
                        callExpr
                          exprList
    decl
      funcDefn
        paramList
        stmtBlock
          stmtList
            stmt
              callStmt
                callExpr
                  exprList
                    expr0
                      expr1
                        expr2
                          expr3
                            callExpr
                              exprList
